java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -280,6 +280,24 @@ CompletableFuture<Boolean> pexpireAt(
 
     // TODO move ScriptingAndFunctionsBaseCommands
     // TODO add note about routing on cluster client
+    /**
+     * Returns the absolute Unix timestamp (since January 1, 1970) at which the given <code>key</code>
+     * will expire.<br>
+     * To get expiration with millisecond precision, use {@link #pexpiretime(String)}.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/commands/expiretime/">redis.io</a> for details.
+     * @param key The <code>key</code> to return its expiration.
+     * @return Expiration timestamp in seconds, <code>-2</code> if <code>key</code> does not exist, or
+     *     <code>-1</code> if <code>key</code> exists but has no associated expire.
```

Comment:
maybe a bit clearer:
```suggestion
     * @param key The <code>key</code> to determine the expiration value of.
     * @return The expiration Unix timestamp in seconds, <code>-2</code> if <code>key</code> does not exist, or
     *     <code>-1</code> if <code>key</code> exists but has no associated expiration.
```